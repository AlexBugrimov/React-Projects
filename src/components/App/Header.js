import React from 'react';
import logo from '../../GitHub-Mark-Light-64px.png';
import './Header.css';

const Header = () => (
  <header className="App-header">
    <img src={logo} className="App-logo" alt="logo" />
    <p>
      My Github Portfolio
    </p>
  </header>
);

export default Header;
