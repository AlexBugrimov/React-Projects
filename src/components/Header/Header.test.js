import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Header from './Header'

describe('the <Header /> component', () => {
  const renderer = new ShallowRenderer();

  it('should render', () => {
    renderer.render(<Header />);
    const result = renderer.getRenderOutput();

    expect(result).toMatchSnapshot();
  })

  it('should render with a dynamic title', () => {
    renderer.render(<Header title="Test Application" />);
    const result = renderer.getRenderOutput();

    expect(result).toMatchSnapshot();
  })
})
