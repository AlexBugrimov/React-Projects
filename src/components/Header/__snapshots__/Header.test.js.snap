// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the <Header /> component should render 1`] = `
<ForwardRef>
  <ForwardRef>
    Hotel Reviews
  </ForwardRef>
</ForwardRef>
`;

exports[`the <Header /> component should render with a dynamic title 1`] = `
<ForwardRef>
  <ForwardRef>
    Test Application
  </ForwardRef>
</ForwardRef>
`;
